ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ModuleContent.vue

```diff
@@ -98,6 +94,15 @@ export default {
       return new RegExp(`^${this.getCurrentCourse?._id}:`)
     }
   },
+  watch: {
+    getSelectedLanguage () {
+      console.log('getSelectedLanguage changed')
```

Comment:
remove `console.log`.
also, can we extract `this.getModuleInfo[this.moduleNum].find(l => l.slug === slug)` into a variable to make it easier to read?

Reply:
i like this idea of DRYing the code